.yarn/patches/source-map-js-npm-1.0.2-ee4f9f9b30.patch

```diff
@@ -0,0 +1,63 @@
+diff --git a/lib/quick-sort.js b/lib/quick-sort.js
```

Comment:
This file is patched because the original file does evaluation (L41) and it is not allowed in extension.